glean-core/tests/datetime.rs

```diff
@@ -39,9 +39,12 @@ fn datetime_serializer_should_correctly_serialize_datetime() {
         );
 
         // `1983-04-13T12:09:14.274+00:00` will be truncated to Minute resolution.
-        let dt = FixedOffset::east(0)
-            .ymd(1983, 4, 13)
-            .and_hms_milli(12, 9, 14, 274);
+        let dt = FixedOffset::east_opt(0)
+            .unwrap()
+            .with_ymd_and_hms(1983, 4, 13, 12, 9, 14)
+            .unwrap()
+            .with_nanosecond(274)
```

Comment:
note: Technically this is 274e6 ns in the original, but the test is still as useful with ns instead of ms.

Reply:
Good catch; I'll fix it.